src/Makefile

```diff
@@ -195,3 +195,19 @@ migrate-canary:
 migrate-local:
 	DJANGO_ENV=local python manage.py migrate
 .PHONY: migrate-local
+
+back-fill-menu-local:
```

Comment:
is this something that needs to be done once?  
Perhaps this is something that could be done within the migration file, so it happens automatically on every database.  Otherwise most peoples local databases won't have this done.